src/software/constants.h

```diff
@@ -50,5 +50,8 @@ const std::string ROBOT_CHIP_PULSE_WIDTH_REDIS_KEY  = "/chip_pulse_width";
 const std::string ROBOT_CURRENT_DRAW_REDIS_KEY      = "/current_draw";
 const std::string ROBOT_BATTERY_VOLTAGE_REDIS_KEY   = "/battery_voltage";
 
-const std::string SSL_ADDRESS          = "224.5.23.2";
-static constexpr unsigned int SSL_PORT = 10020;
+const std::string SSL_VISION_ADDRESS          = "224.5.23.2";
+static constexpr unsigned int SSL_VISION_PORT = 10006;
```

Comment:
Should the vision port not remain at `10020`? That's atleast what we've been using during field testing at EDC, not sure what port we used at robocup though. This should probably be whatever is used at robocup.

Reply:
Checked the standard network parameters https://ssl.robocup.org/league-software/#:~:text=Simulation%20Protocol.-,Standard,-Network%20Parameters 10006 is the robocup port, 10020 is the default for er-force. changing it back to 10020

Reply:
Good find with the document, I hadn't seen it before. In terms of the port, imo we should use whatever is standard to Robocup to avoid confusions in the future and if it is possible, update ErForce sim to reflect that. Though having a quick look around the repo, I wasn't able to find where ErForce defines the vision port.

Reply:
Changed it back to 10006. Didnt see it in the repo either, it defaults to 10020 according to people in the SSL Discord. Might be in the autoref code or smth not sure.